.github/CONTRIBUTING.md

```diff
@@ -28,12 +28,11 @@ This is the workflow for contributing changes to Emogrifier:
    [coding style](#coding-style) as the rest of the project.
 7. Add a changelog entry.
 8. [Commit](#git-commits) and push your changes.
-9. [Create a pull request](https://help.github.com/articles/about-pull-requests/)
-   for your changes. Check that the Travis build is green. (If it is not, fix
-   the problems listed by Travis.)
-10. [Request a review](https://help.github.com/articles/about-pull-request-reviews/)
-    from @oliverklee.
-11. Together with him, polish your changes until they are ready to be merged.
+9. [Create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+   for your changes. Check that the CI build is green. (If it is not, fix the
+   problems listed.)
```

Comment:
First-time contributors will need to wait for an admin to allow the CI build to run.  I think it's worth noting such here.